app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    original_file = command.split()[1]
+    new_file = command.split()[2]
+
+    if original_file != new_file:
```

Comment:
also add check for this case  
<img width="956" alt="image" src="https://user-images.githubusercontent.com/93856423/220702963-65270573-dc66-4e77-adde-2667252a9dc7.png">